db/schema.rb

```diff
@@ -1532,9 +1532,8 @@
   end
 
   create_table "va_caution_flags", id: :serial, force: :cascade do |t|
-    t.string "institution_id"
+    t.string "facility_code", null: false
     t.string "institution_name"
-    t.integer "school_system_code"
```

Comment:
Is there a migration for these changes? I dont think you want to be altering the schema.rb directly.

Reply:
i ran migrate locally and these occurred, resetting file to what is in master branch